### ISSUE TITLE
Unable to expose ports to outside on https://labs.play-with-k8s.com

### ISSUE THREAD

**Issue by basilky** [play-with-docker/play-with-docker#588]
The services exposed from the cluster is not accessible from outside. @marcosnils I created a cluster similar to the one you shown [here](https://github.com/play-with-docker/play-with-docker/issues/345#issuecomment-508942080). But I'm getting gateway timeout error when accessing the service. It would be great to have this functionality working.

**Comment by nageshbandi** [comment]
yes, Not able to access the application with URL and exposed port number. Please help some once.

**Comment by prakashpanjwani** [comment]
Not able to access the application with URL and exposed port number. Please help some once.

**Comment by gertsog333** [comment]
Is It not working again. Could you please help ? 
Not able to access the application with URL and exposed port number. Could you please help ?
Thanks
